### PR TITLE
Range error fix

### PR DESCRIPTION
**Pull request by htkcodes**
Fixes the "RangeError: Maximum call stack size exceeded" error.

Tested with ~174k line file.

### CODE REVIEW

**Comment by htkcodes** [comment]
fixes #18

File: src/graphs.js [hunk 1]
```diff
@@ -67,11 +67,34 @@ function set_graph_0_pie(totals_data, names) {
...
+}
+
 
 function set_graph_0_whisk(just_data, elem_id, color, description) {
     // Stats
-    let min = Math.min(...just_data);
-    let max = Math.max(...just_data);
+    let min = getMin(just_data)
+    let max = getMax(just_data)
+    console.log(min)
```
Comment (BryceStevenWilley) [inline src/graphs.js]: Remove this extra log, I try not to keep feature-specfic debug logs in master.

**Review by BryceStevenWilley — APPROVED** [review]
LGTM with that small comment!  Thanks for all the help @htkcodes!

File: src/graphs.js (hunk 1 above)
Comment (htkcodes) [reply src/graphs.js]: removed. :)

**Comment by BryceStevenWilley** [comment]
Perfect, merged!

**Pull request merged**